src/redux/search.js

```diff
@@ -5,6 +5,7 @@ import 'rxjs/add/observable/from';
 import 'rxjs/add/operator/switchMap';
 import 'rxjs/add/operator/pluck';
 import 'rxjs/add/operator/catch';
+import _ from 'lodash';
```

Comment:
We should actually just import `merge` here since that's all that's used in this file.

```js
import merge from 'lodash/merge';
```